recipes/pysdl2/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "PySDL2" %}
+{% set version = "0.9.4" %}
+{% set md5 = "720b33ed048c169643406ae39ca4dd8d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn:  {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+  patches:
+    - locate_sdl.patch
```

Comment:
Thanks for reviewing! I'm not exactly sure what you mean. I included this patch from the beginning in the main recipe. I cannot place the patch in the recipe itself and have to add it as a separate PR? I'm a bit confused about what I have to do now. I have to remove the patch here, commit, push, and then include the patch and send it as a separate pull request?